packages/template-mrt-reference-app/app/isolation-actions.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const {LambdaClient, InvokeCommand} = require('@aws-sdk/client-lambda')
+const {S3Client, GetObjectCommand} = require('@aws-sdk/client-s3')
+const {CloudWatchLogsClient, PutLogEventsCommand} = require('@aws-sdk/client-cloudwatch-logs')
+
+export const isolationOriginLambdaTest = async (input) => {
+    const client = new LambdaClient()
+    try {
+        await client.send(new InvokeCommand(input))
+    } catch (e) {
+        if (e.name === 'AccessDeniedException') {
+            return true
+        }
+        console.error(e)
+    }
+    console.error('Lambda isolation test failed!')
```

Comment:
Are you simply expecting this to be in your test run console STDOUT? Some execution contexts for tests can swallow this kind of thing. You might want to consider `expect(console, 'error').notToHaveBeenCalled()` or something along those lines.

Reply:
These tests aren't primarily meant to be executed as jest unit tests but as integration tests as a deployed app. The console.error is just to add more information in the SSR logs when it fails once deployed.